### ISSUE TITLE
Replace Moberg with local event log

### ISSUE THREAD

**Issue by hraberg** [xtdb/xtdb#322]
We should consider removing Moberg and just use JDBC with either H2 or SQLite for the standalone mode. Could help solve #280 and maybe also #321 (assuming this is a timing issue in related to the log). Would remove a moving piece and some code to maintain.

A drawback is that the JDBC log is still very fresh, and not tuned or tested in anger. It also has no indexes yet for example.

**Comment by hraberg** [comment]
Obviously depends on #270.

**Comment by hraberg** [comment]
In practice this is mainly a question about rejigging start-standalone-node to use some of the JDBC properties instead of the event-log ones.

If someone is actively using Moberg in (semi) production there would need to be a migration path. This is itself a reason to do this sooner rather than later if we decide to.

We could also decide to keep Moberg around for a bit and deprecate it and potentially write a simple tool that can use the public API to migrate the data from one Crux instance to another.

**Comment by hraberg** [comment]
As a side effect of this, it would be good to rejig the polling and consumption code so it can be shared between Kafka and JDBC, but that's likely a separate card.

**Comment by hraberg** [comment]
Note that an unintended side effect of this would have standalone system not being possible to run with `crux-core` on its own. It would also have to depend on h2 (or less likely, SQLite).
